CHANGELOG.md

```diff
@@ -1,6 +1,58 @@
 # Release Notes
 
-## Platform Version 0.10.8 - UNRELEASED
+## Platform Version 0.10.9 - UNRELEASED
```

Comment:
Did git cliff add this UNRELEASED header?  I think we should leave it out for the release and put it back in post release w/ the version bump.

Reply:
yes, cliff did add it. just removed